python/integration_tests/worker.py

```diff
@@ -87,3 +89,44 @@ def process_task(self, activation: TaskActivation) -> TaskActivation | None:
             status=TASK_ACTIVATION_STATUS_COMPLETE,
             fetch_next_task=FetchNextTask(namespace=self._namespace),
         )
+
+
+class ConfigurableTaskWorker:
```

Comment:
Perhaps in the future we can replace `SimpleTaskWorker` usage with this? If both timeout and failure rates are 0, they should be functionally the same.

Reply:
Makes sense to me.